### ISSUE TITLE
Unused language strings should be removed from language files

### ISSUE THREAD

**Issue by horus68** [salesagility/SuiteCRM#2444]
#### Issue
TCPDF examples should be removed from translations
'LBL_FONT_LIST_CIDINFO_INFO' from Configurator/language https://github.com/salesagility/SuiteCRM/blob/hotfix/modules/Configurator/language/en_us.lang.php#L336
is a difficult string to translate and easy to cause parsing errors. Actually its almost a code example and it should be just the link for more infos and without the examples
#### Expected Behavior

A simple line with link for more information
#### Actual Behavior

Strings should be removed

```
    'LBL_FONT_LIST_CIDINFO_INFO' => 'Examples :
<ul><li>.
Chinese Traditional :<br>.
<pre>\$enc=\'UniCNS-UTF16-H\';<br>.
\$cidinfo=array(\'Registry\'=>\'Adobe\', \'Ordering\'=>\'CNS1\',\'Supplement\'=>0);<br>.
include(\'include/tcpdf/fonts/uni2cid_ac15.php\');</pre>.
</li><li>.
Chinese Simplified :<br>.
<pre>\$enc=\'UniGB-UTF16-H\';<br>.
\$cidinfo=array(\'Registry\'=>\'Adobe\', \'Ordering\'=>\'GB1\',\'Supplement\'=>2);<br>.
include(\'include/tcpdf/fonts/uni2cid_ag15.php\');</pre>.
</li><li>.
Korean :<br>.
<pre>\$enc=\'UniKS-UTF16-H\';<br>.
\$cidinfo=array(\'Registry\'=>\'Adobe\', \'Ordering\'=>\'Korea1\',\'Supplement\'=>0);<br>.
include(\'include/tcpdf/fonts/uni2cid_ak12.php\');</pre>.
</li><li>.
Japanese :<br>.
<pre>\$enc=\'UniJIS-UTF16-H\';<br>.
\$cidinfo=array(\'Registry\'=>\'Adobe\', \'Ordering\'=>\'Japan1\',\'Supplement\'=>5);<br>.
include(\'include/tcpdf/fonts/uni2cid_aj16.php\');</pre>.
</li></ul>.
More help : www.tcpdf.org',
```
#### Possible Fix

`
    'LBL_FONT_LIST_CIDINFO_INFO' => 'For examples and more help : www.tcpdf.org',
`
#### Context
There is no point to have code examples in translations for different languages.
Else its easy to break the system with parsing errors from the translations (unescaped braces and so on)

#### Your Environment

- SuiteCRM Version used: SuiteCRM 7.7.5

**Comment by likhobory** [comment]
And this is just one example of more than 300 unused labels.
All same PDF labels have been used in SugarCRM PRO (!) only and should be removed from translations.
https://github.com/likhobory/SuiteCRM7RU/blob/ver.7.6.1/modules/Configurator/language/ru_ru.lang.php#L206-L396

**Comment by horus68** [comment]
@likhobory lets make a PR for those strings?

**Comment by likhobory** [comment]
I'll add several issues with unused labels for each module.

**Comment by ijdavie** [comment]
@horus68, I have raised this as a suggestion, partly from what we have seen this links to a page which looks like it is not used within SuiteCRM and could be left over from Sugars Core.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Unused language strings should be removed from language files](https://trello.com/c/Y9aV2vzp/63-unused-language-strings-should-be-removed-from-language-files)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
